src/esp/nav/PathFinder.cpp

```diff
@@ -1329,7 +1343,7 @@ bool PathFinder::build(const NavMeshSettings& bs,
   return pimpl_->build(bs, mesh);
 }
 
-vec3f PathFinder::getRandomNavigablePoint() {
+vec3f PathFinder::getRandomNavigablePoint(const int maxTries /*= 10*/) {
   return pimpl_->getRandomNavigablePoint();
```

Comment:
Need to pass the variable 🙂

Reply:
I'm glad my clang-tidy checks caught this bug appearing on the exact same class. 👍

Reply:
Looks like I forgot to check if this would compile locally... this is exactly why I didn't put a default value in the pimpl header...

Reply:
@erikwijmans Even if you did, we now have clang-tidy checks for unused values. 👍